MAUI/Kanban-Board/Cards.md

```diff
@@ -21,16 +21,16 @@ The default elements of a card can be customized using the below properties of [
 
 {% highlight C# %}
 
-    new KanbanModel()
```

Comment:
@EzhilarasanElangovan31 line 19, update as  -> Used to specify the indicator color of a card.

Reply:
Updated